book_source/03_topical_pages/09_standalone_tools.Rmd

```diff
@@ -159,83 +159,182 @@ Processing functions currently available are,
 
-3. **Install the Python dependencies**. Using this module requires Python3 and the package manager pip to be installed in your system. 
-   To install the additional Python dependencies required, 
-a. Navigate to `pecan/modules/data.remote/inst` If you are inside the pecan directory, this can be done by,
+3. **Install the RpTools package**. Python codes required by this module are stored in a Python package named "RpTools" using this requires Python3 and the package manager pip3 to be installed in your system. 
+   To install the package,
+a. Navigate to `pecan/modules/data.remote/inst/RpTools` If you are inside the pecan directory, this can be done by,
 ```bash
-cd modules/data.remote/inst
+cd modules/data.remote/inst/RpTools
 ```
-b. Use pip to install the dependencies.
+b. Use pip3 to install the package. "-e" flag is used to install the package in an editable or develop mode, so that changes made to the code get updated in the package without reinstalling.
 ```bash
-pip install -r requirements.txt
+pip3 install -e .
 ```
 4. **Authenticate GEE**. The GEE API needs to be authenticated using your credentials. The credentials will be stored locally on your system. This can be done by,
 ```bash
 #this will open a browser and ask you to sign in with the Google account registered for GEE
 earthengine authenticate
 ```
-5. **Save the Earthdata credentials (optional)**. If you do not wish to enter your credentials every time you use AppEEARS, you may save your username and password inside a JSON file and then pass its file path as an argument in `remote_process` 
+5. **Save the Earthdata credentials**. If you wish to use AppEEARS you will have to store your username and password inside a JSON file and then pass its file path as an argument in `remote_process` 
 
 #### Usage guide:
-This module contains GEE and other processing functions which retrieve data from a specific dataset and source. For example, `gee2pecan_s2()` downloads bands from Sentinel 2 using GEE and `bands2lai_snap()` uses the downloaded bands to compute LAI using the SNAP algorithm. These functions can be used independently but the recommended way is to use `remote_process()` which is a main function that controls all the individual functions to create an organized way of downloading and handling remote sensing data in PEcAn. 
+This module contains GEE and other processing functions which retrieve data from a specific dataset and source. For example, `gee2pecan_s2()` downloads bands from Sentinel 2 using GEE and `bands2lai_snap()` uses the downloaded bands to compute LAI using the SNAP algorithm. These functions can be used independently but the recommended way is to use `remote_process`which is the main function that controls all the individual functions to create an organized way of downloading and storing remote sensing data in PEcAn. 
+
```

Comment:
if you have time, it would be great if you could include a workflow scheme around here that explains the high-level functionality, which function calls what, which decisions are made...